cmd/core-data/res/configuration.toml

```diff
@@ -17,6 +17,14 @@ StartupMsg = "This is the Core Data Microservice"
 MaxResultCount = 1024
 MaxRequestSize = 0 # Not curently used. Defines the maximum size of http request body in bytes
 RequestTimeout = "5s"
+  [Service.CORSConfiguration]
```

Comment:
Need this for all the other Core/Support services also.

Reply:
AllowedHeaders should include by default:
Authorization, Accept, Accept-Language, Content-Language, Content-Type
The last four are the "CORS-safelisted" headers and should always be present
https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
we are adding Authorization.

ExposedHeaders should include
Cache-Control, Content-Language, Content-Length, Content-Type, Expires, Last-Modified, Pragma
(as per https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_response_header)

Reply:
@lenny-intel Do we also need to include the correlation ids and such in the above list?